src/main/kotlin/auth/config/adminId/LoginAdminIdArgumentResolver.kt

```diff
@@ -0,0 +1,29 @@
+package uket.auth.config.adminId
+
+import org.springframework.core.MethodParameter
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.stereotype.Component
+import org.springframework.web.bind.support.WebDataBinderFactory
+import org.springframework.web.context.request.NativeWebRequest
+import org.springframework.web.method.support.HandlerMethodArgumentResolver
+import org.springframework.web.method.support.ModelAndViewContainer
+
+@Component
+class LoginAdminIdArgumentResolver : HandlerMethodArgumentResolver {
+    override fun supportsParameter(parameter: MethodParameter): Boolean {
+        return parameter.hasParameterAnnotation(LoginAdminId::class.java)
+    }
+
+    override fun resolveArgument(
+        parameter: MethodParameter,
+        mavContainer: ModelAndViewContainer?,
+        webRequest: NativeWebRequest,
+        binderFactory: WebDataBinderFactory?,
+    ): Long {
+        val role: String = SecurityContextHolder.getContext().authentication.authorities.first().authority
+        check(role == "ADMIN") { "잘못된 토큰 전달입니다." }
```

Comment:
역할이 더 추가될 것 같지는 않지만, any를 활용해보는 건 어떨까여

```suggestion
        val isAdmin = SecurityContextHolder.getContext().authentication.authorities.any { it.authority == "ADMIN" }
        check(isAdmin) { "잘못된 토큰 전달입니다." }
```

Reply:
좋은 거 같습니다! 맞춰서 수정해보도록 하겠습니다.